app.rb

```diff
@@ -18,11 +20,19 @@ def initialize
              end
     @albums = []
     @games = []
+
+    @books = []
+    @music_albums = []
```

Comment:
I think It is a good idea to erase lines 23, 24, and 25 cause are overridden the values loaded from the JSON files.

Reply:
Thank you for highlighting this